manual/manual/macros.hva

```diff
@@ -23,12 +23,12 @@
 
 % Compact layout
 \newstyle{body}{
-  max-width:850px;
+  max-width:750px;
   width: 85\%;
   margin: auto;
   background: \#f7f7f7;
   margin-top: 80px;
-  font-size: 19px
+  font-size: 1rem;
 }
```

Comment:
18px would be 1.125rem
19px would be 1.1875rem

I'd suggest going for 1.125rem unless this is a problem. Now that we're using relative sizes, people can adjust their browsers to suit.

Reply:
@dbuenzli, @pmetzger Seems everyone has strong convictions about this. I'm tempted to revert this piece to keep the original manual's formatting until we can come to consensus. Let's remember that we're talking about a couple pixels and we're all on the same side :)  A noble debate nonetheless!

Reply:
Oh my I thought that discussion was over. Again, respect the user's defaults.

Reply:
Hey, my suggestion was smaller than the original 19px! That's a compromise.

If you don't want to do anything, though, leave it 1rem explicitly and we'll reconsider later.